action/protocol/staking/handler_stake_migrate.go

```diff
@@ -162,7 +165,7 @@ func (p *Protocol) withdrawBucket(ctx context.Context, withdrawer *state.Account
 	}, nil
 }
 
-func (p *Protocol) ConstructExecution(ctx context.Context, act *action.MigrateStake, sr protocol.StateReader) (*action.Execution, error) {
+func (p *Protocol) ConstructExecution(ctx context.Context, act *action.MigrateStake, nonce, gas uint64, gasPrice *big.Int, sr protocol.StateReader) (*action.Execution, error) {
```

Comment:
we should get `nonce` and `gasPrice` from ActionCtx, the extra params not needed